README.md

```diff
@@ -180,6 +180,7 @@ DevOps is the combination of cultural philosophies, practices, and tools that in
 - [Digger](https://digger.dev) - Open Source Infrastructure as Code management tool that runs within your CI/CD system.
 - [Deployment.io](https://deployment.io) - DevOps co-pilot for developers to automate deployments to AWS.
 - [Terrateam](https://terrateam.io) - Open-source alternative to Terraform Cloud/Enterprise, GitOps-first with native GitHub integration and designed for scale, security, and reliability.
+- [vCluster](https://vcluster.sh/)- A open source project that helps you create virtual clusters
```

Comment:
Hi,

Can you move it to `Applications Platforms` section?

Reply:
Sure done @wmariuss :)